### ISSUE TITLE
IMOS:  check_longitude_variable + check_latitude_variable - missing attributes, but error message don't match

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#39]
ncdump output

```
double LATITUDE(time) ;                                                                                       |                                                      present                                                         
                LATITUDE:long_name = "LATITUDE" ;                                                                     |        valid_min                      :3:     0/ 1 :                                                                 
                LATITUDE:standard_name = "LATITUDE" ;                                                                 |            check_min_value            :3:     0/ 1 : Variable attribute is not                                       
                LATITUDE:units = "degrees_north" ;                                                                    |                                                      present                                                         
                LATITUDE:_FillValue = 999999. 

 double LONGITUDE(time) ;                                                                                      |            check_max_value            :3:     0/ 1 : doesn't match value pair                                        
                LONGITUDE:long_name = "LONGITUDE" ;                                                                   |                                                      (0, 360) or (-180, 180)                                         
                LONGITUDE:standard_name = "LONGITUDE" ;                                                               |        valid_min                      :3:     0/ 1 :                                                                 
                LONGITUDE:units = "degrees_east" ;                                                                    |            check_min_value            :3:     0/ 1 : doesn't match value pair                                        
                LONGITUDE:_FillValue = 999999. ;       

```

As can be seen above, there is no sign of valid_max and valid_min attributes. However, the imos checker says for **LONGITUDE**:

```
       valid_min                      :3:     0/ 1 : 
          check_min_value            :3:     0/ 1 : doesn't match value pair 
                                                     (0, 360) or (-180, 180)
```

but for **LATITUDE** the checker only says 

```
       valid_max                      :3:     0/ 1 :                                   
            check_max_value            :3:     0/ 1 : Variable attribute is not    present
```

A bit weird, there shouldn't be any difference between the two ...

tested file :
http://thredds.aodn.org.au/thredds/catalog/IMOS/SOOP/SOOP-TRV/VMQ9273_Solander/By_Cruise/Cruise_START-20150105T073551Z_END-20150106T024945Z/salinity/catalog.html?dataset=IMOS/SOOP/SOOP-TRV/VMQ9273_Solander/By_Cruise/Cruise_START-20150105T073551Z_END-20150106T024945Z/salinity/IMOS_SOOP-TRV_S_20150105T073551Z_VMQ9273_FV01_END-20150106T024945Z_C-20150106T190156Z.nc

**Comment by fxmzb123** [comment]
@lbesnard fixed and please see pull request #44

**Comment by lbesnard** [comment]
thx, works fine
